source/includes/_sets.md

```diff
@@ -232,13 +237,18 @@ This endpoint updates a specific TestSet.
 Available parameters | Description |
 --------- | ------- |
 data/attributes/name | name |
-data/attributes/assigned-to-id | user assigned-to id (not Display ID) - [users list](#users)  |
+data/attributes/assigned-to-id | user or group assigned-to id (not Display ID) - [users list](#users)  | false |
+data/attributes/assigned-to-type | assigned-to type (user or group) | false |
 data/attributes/planned-execution | date field of planned-execution |
 data/attributes/version | string of TestSet version |
 data/attributes/priority | string of TestSet priority |
 data/attributes/custom-fields | a hash of custom-fields with their value |
 data/attributes/tags | an array of tags | false |
 
+* assigned-to-id and assigned-to-type are not required. If only assigned-to-id is passed, then type will be considered as user.
+So when assign to a group is wanted, assigned-to-type group must be passed.
```

Comment:
this is re-written in duplicated places.
maybe it should be placed somewhere on top, talking about assigned-to field, and how to approach it?
and then link to [see more here]

Reply:
do you mean like for the > API rate limit

Reply:
done